main.go

```diff
@@ -483,7 +483,7 @@ func getMinorUpdateVersion(path string) (string, error) {
 	result := results[0]
 
 	if result.Error != nil {
-		return "", fmt.Errorf("error getting module info: %s", path, result.Error.Err)
+		return "", fmt.Errorf("error getting module info: %s", result.Error.Err)
```

Comment:
This is now a hard error. I just removed the extra parameter - not sure if you'd prefer to work that parameter into the error.

Reply:
I will probably try to work that parameter into the error, but I can do that post-merge. Thanks!!

Reply:
Actually, nevermind. It looks like the `path` is already being added to the error message after the error is returned from `getMinorUpdateVersion`, when it's wrapped again [here](https://github.com/nathanjcochran/upgrade/blob/master/main.go#L428). I bet the `path` variable you removed was accidentally left-over from before I started wrapping the errors in a higher-level error like that. Thanks for fixing that!